Simple-Projects/Simple_Projects/Basic/Complex/Complex.cpp

```diff
@@ -1,10 +1,10 @@
+#include <ostream>
 #include "Complex.h"
 
-Complex::Complex(double real, double imaginary): real(0), imaginary(0) {
-    this->real = real;
-    this->imaginary = imaginary;
+Complex::Complex(double real, double imaginary): real(real), imaginary(imaginary) {
```

Comment:
Initalizing members with temporary values is redundant, why not initialize them with the proper values right away?